app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -1,51 +1,56 @@
 import React from "react";
 
-import dayjs from "dayjs";
 import { MenuVertical, Clock } from "neetoicons";
 import { Typography, Dropdown, Tooltip, Avatar, Tag } from "neetoui";
 
-const { Menu, MenuItem } = Dropdown;
+import { formatCreatedAt, calculateCreatedAgo } from "./utils";
 
-const Note = ({ note }) => (
-  <div className="mb-3 w-full border border-gray-300 bg-white p-4 shadow-md dark:border-gray-700 dark:bg-gray-800 ">
-    <div className="flex justify-between">
-      <Typography style="h4" weight="bold">
-        {note.title}
+const Note = ({ note, setShowDeleteAlert, setSelectedNoteId }) => {
+  const { Menu, MenuItem } = Dropdown;
```

Comment:
Let's keep this outside the component. We don't need this statement to run again and again whenever the component re-renders.